### PR TITLE
Added `transitions` tests

### LINKED ISSUE #1465: Enhance Test Coverage for `@yamada-ui/transitions`
package: @yamada-ui/transitions

### Description
`@yamada-ui/transitions` has a test coverage of less than 90%. We will enhance it to above 90%.

### Target files
- [packages/components/transitions/src/slide-fade.tsx](https://app.codecov.io/gh/yamada-ui/yamada-ui/blob/main/packages/components/transitions/src/slide-fade.tsx)
  - [L42](https://app.codecov.io/gh/yamada-ui/yamada-ui/blob/main/packages/components/transitions/src/slide-fade.tsx#L42)
  - [L58](https://app.codecov.io/gh/yamada-ui/yamada-ui/blob/main/packages/components/transitions/src/slide-fade.tsx#L58)

### Problem Statement/Justification
Testing is essential to ensure quality when operating a package.

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.

### PR DESCRIPTION
**Pull request by 108yen**
Closes #1465

## Description

Added `transitions` tests for enhance test coverage.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Comment by koralle** [comment]
@108yen 

You can use `.toBeNull()` instead of `.not.toBeInTheDocument()`

**Comment by 108yen** [comment]
> @108yen
> 
> You can use `.toBeNull()` instead of `.not.toBeInTheDocument()`.

Fixed!

File: packages/components/transitions/tests/slide-fade.test.tsx [hunk 1]
```diff
@@ -1,38 +1,121 @@
-import { render } from "@yamada-ui/test"
+import { waitFor } from "@testing-library/react"
```
Comment (setodeve) [inline packages/components/transitions/tests/slide-fade.test.tsx:1]: `waitFor` can be called from `@yamada-ui/test`.

**Review by setodeve — CHANGES_REQUESTED** [review]

File: packages/components/transitions/tests/slide-fade.test.tsx (hunk 1 above)
Comment (108yen) [reply packages/components/transitions/tests/slide-fade.test.tsx]: @setodeve

Fixed!

**Review by setodeve — APPROVED** [review]

**Pull request merged**